### PR TITLE
Initial version of the quickstart guide document.

### PR DESCRIPTION
**Pull request by rossphilipson**
Initial version of the quickstart guide document.

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+000005b0  b2 29 3f 3c da 25 4a 78  61 be 76 91 3e 06 f9 5d  |.)?<.%Jxa.v.>..]|
+000005c0  7d 6b 0d 75 6b 30 74 0c  26 b2 76 96 1e 60 19 a5  |}k.uk0t.&.v..`..|
+000005d0  18 00 00 00 4d 65 61 73  75 72 65 64 20 55 45 46  |....Measured UEF|
+000005e0  49 20 6d 65 6d 6f 72 79  20 6d 61 70 11 00 00 00  |I memory map....|
+000005f0  04 05 00 00 01 00 00 00  0b 00 00 00 00 00 00 00  |................|
+00000600  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+*
+00008000
+```
+The final measurements starting with with the description "Measured..." are put in the
```
Comment (stacktrust) [inline QUICKSTART.md:134]: s/with with/with

File: QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+# Quick Start Guide
+
+A quick start guide to getting a system running with the latest TrenchBoot
+Secure Launch bits. Note that this is a bare bones document meant to help
+someone get up and running with Secure Launch. It does not contain detailed
+descriptions of all the technologies and terminology involved in doing a
+secure late launch. The repository this document resides in contains a plethora
```
Comment (pietrushnic) [inline QUICKSTART.md:7]: Why sometime we write Secure Launch and sometimes with small letters, like "secure late launch"?

File: QUICKSTART.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+`# grep smx /proc/cpuinfo`
+
+## Linux
+
+TrenchBoot and the Secure Launch feature are an active project that is in the
+process of being upstreamed to the Linux project.
+
+The following repository and branch have the latest release of the Secure
+Launch feature. This is a vanilla Linux kernel based of a torvolds/master branch
```
Comment (stacktrust) [inline QUICKSTART.md:31]: s/of/off/
s/torvolds/torvalds/

File: QUICKSTART.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+enabled. The details on how to do this are system specific. To see if the CPU
+supports TXT, run the following (SMX (Safe Mode Extensions) indicates the CPU
+does support TXT):
+
+`# grep smx /proc/cpuinfo`
+
+## Linux
+
+TrenchBoot and the Secure Launch feature are an active project that is in the
+process of being upstreamed to the Linux project.
```
Comment (stacktrust) [inline QUICKSTART.md:28]: How about:

TrenchBoot is an active open-source project for system launch integrity, from which the Secure Launch feature is being upstreamed to the Linux kernel.

File: QUICKSTART.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+000005e0  49 20 6d 65 6d 6f 72 79  20 6d 61 70 11 00 00 00  |I memory map....|
+000005f0  04 05 00 00 01 00 00 00  0b 00 00 00 00 00 00 00  |................|
+00000600  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+*
+00008000
+```
+The final measurements starting with with the description "Measured..." are put in the
+log by the Secure Launch kernel code after successfully running. During a poweroff,
+restart or a kexec of another kernel, the following log lines will show TXT being
+properly diabled and safely locked:
```
Comment (stacktrust) [inline QUICKSTART.md:137]: s/diabled/disabled/

File: QUICKSTART.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
+# Quick Start Guide
+
+A quick start guide to getting a system running with the latest TrenchBoot
```
Comment (stacktrust) [inline QUICKSTART.md:3]: s/system/Linux system/
s/TrenchBoot Secure Launch bits/Secure Launch bits from TrenchBoot/

**Review by stacktrust — CHANGES_REQUESTED** [review]
How about adding a Community section at the end with links to:

- TrenchBoot presentations web page
- TrenchBoot docs web page
- TrenchBoot Matrix channel

**Comment by rossphilipson** [comment]
> How about adding a Community section at the end with links to:
> 
> * TrenchBoot presentations web page
> * TrenchBoot docs web page
> * TrenchBoot Matrix channel

I was trying to limit the scope of the document to be as simple as possible. Thus my disclaimer at the beginning about it not covering everything.

File: QUICKSTART.md (hunk 1 above)
Comment (rossphilipson) [reply QUICKSTART.md:134]: Will fix

File: QUICKSTART.md (hunk 3 above)
Comment (rossphilipson) [reply QUICKSTART.md:31]: Will fix

File: QUICKSTART.md (hunk 4 above)
Comment (rossphilipson) [reply QUICKSTART.md:28]: Sure can do

File: QUICKSTART.md (hunk 5 above)
Comment (rossphilipson) [reply QUICKSTART.md:137]: Will fix

File: QUICKSTART.md (hunk 6 above)
Comment (rossphilipson) [reply QUICKSTART.md:3]: Sure

File: QUICKSTART.md (hunk 2 above)
Comment (rossphilipson) [reply QUICKSTART.md:7]: I guess I was trying to say something else but it ended up being unclear. I can just change it.

**Comment by stacktrust** [comment]
> > How about adding a Community section at the end with links to:
> 
> I was trying to limit the scope of the document to be as simple as possible. Thus my disclaimer at the beginning about it not covering everything.

Since many topics will be missing, by definition, there should be at least one sentence about where to go for more information, e.g

For topics not addressed by this document, please contact TrenchBoot developers via https://trenchboot.org/community.

**Review by stacktrust — APPROVED** [review]

File: QUICKSTART.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+An Intel system (desktop, server, laptop) needs to be a vPro SKU in order to
+have TXT support available. Generally speaking, vPro systems will advertise this
+with a sticker somewhere on the unit. Intel TXT support usually needs to be
+enabled in the firmware setup program. It depends on both the TPM and VTd being
+enabled. The details on how to do this are system specific. To see if the CPU
+supports TXT, run the following (SMX (Safe Mode Extensions) indicates the CPU
+does support TXT):
+
+`# grep smx /proc/cpuinfo`
+
```
[the remark this answers is not in the document]
Comment (rossphilipson) [reply QUICKSTART.md:29]: Ok I will add something along those lines

**Review by dpsmith — APPROVED** [review]

**Pull request merged**